fastlane/actions/activate_rogue_team.rb

```diff
@@ -53,7 +53,7 @@ def self.available_options
           FastlaneCore::ConfigItem.new(key: :bundle_id,
                                        default_value: 'com.volz.drew.aao.rogue'),
           FastlaneCore::ConfigItem.new(key: :profile_name,
-                                       default_value: 'match Development com.volz.drew.aao.rogue'),
+                                       default_value: 'match Development com.volz.drew.aao.rogue')
```

Comment:
So I really like trailing commas. Can we make rubocop add them for us?

Reply:
Does ESLint allow this?

Reply:
Yes, and prettier adds them all for us in our JS code.

Reply:
[This entry in the style guide](https://github.com/bbatsov/ruby-style-guide#no-trailing-array-commas) is why I did all of these.

Reply:
Yeah, I think I remember reading that back when I was trying to learn ruby. I just disagree :p 

I like trailing commas, because they let me add new items to the collection without highlighting extra lines in the diff.